docs/extend/extend-apiml/onboard-wizard.md

```diff
@@ -1,12 +1,18 @@
 # Onboarding a REST API service with the YAML Wizard
-## Description
-The purpose of this wizard is to simplify the process of onboarding new REST API services to the Zowe API Mediation Layer. It offers a walkthrough of the required steps to create a correct configuration file which is used to set the application properties and eureka metadata.
+
+As an API developer, you can use the Yaml Onboarding Wizard to simplify the process of onboarding new REST API services to the Zowe API Mediation Layer. The wizard offers a walkthrough of the required steps to create a correct configuration file which is used to set the application properties and Eureka metadata.
+
 ## Onboarding your REST service with the Wizard
-1. Click on the 'Onboard New API' dropdown located in the navbar  
+
+The following procedure describes how to onboard your REST service with the Wizard.
+
+**Follow these steps:**
+
```

Comment:
I thought this article was going to include how to install the wizard...

Reply:
The wizard is embedded in the webpage. The user would just need to click the dropdown, and the wizard will pop up as a modal dialog.

Reply:
Ok. What webpage?

Reply:
It's part of the API catalog. We are working on localhost so we don't know, but should we find out the link to the deployed version and paste it there?

Reply:
Yes. Let's be specific when we have the deployed version.